app/src/main/res/layout/logins_item.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's undo this. This is unfortunate one of the problem with the autoformat.

Reply:
done